EmpleoDotNet/ViewModel/NewJobOpportunityViewModel.cs

```diff
@@ -60,6 +66,18 @@ public Core.Domain.JobOpportunity ToEntity()
                 IsRemote = IsRemote
             };
 
+            if (!string.IsNullOrWhiteSpace(LocationName) &&
+                !string.IsNullOrWhiteSpace(LocationPlaceId))
+            {
+                entity.JobOpportunityLocation = new JobOpportunityLocation
+                {
+                    Latitude = LocationLatitude,
+                    Longitude = LocationLongitude,
+                    Name = LocationName,
+                    PlaceId = LocationPlaceId
+                };
+            }
+
             return entity;
```

Comment:
I just saw the same code twice. We could abstract this new code into a class and use it in the viewmodels that need it.